spec/features/feedback_spec.rb

```diff
@@ -80,6 +80,8 @@
       end
 
       background do
+        application.authority.update_attribute(:write_to_councillors_enabled, true)
```

Comment:
I've got it.

Speaking of this, I don't really understand what [`#update_attribute`](http://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-update_attribute) is for. The implementation is quite different from `#update` and I don't really see why.